conf/gulp-tasks/library.gulpfile.js

```diff
@@ -21,9 +21,21 @@ const uglify = require('gulp-uglify-es').default;
 const NAMESPACE = 'ANSWERS';
 
 function getLibVersion () {
-  return require('child_process')
-    .execSync('git describe --tags')
-    .toString().trim();
+  try {
+    let insideWorkTree = require('child_process')
+      .execSync('git rev-parse --is-inside-work-tree 2>/dev/null')
+      .toString().trim();
+    if (insideWorkTree) {
```

Comment:
Is insideWorkTree a string? Do we need to convert it to a boolean since "false" is truthy?

Reply:
My worry was that this would be too restrictive for getting a valid version. I have only personally seen the value `true` and then if we were not in a git repository, we did not get a value back and instead just got a fatal error (explaining the catch). I didn't have any luck finding all the possible values of the function, but I can keep digging around here

https://github.com/git/git/blob/3bab5d56259722843359702bc27111475437ad2a/setup.c#L403

Reply:
Underlying functions return booleans and fatals, will check specifically for true here